apps/condo/migrations/20210813135914-0042_auto_20210813_1359.js

```diff
@@ -0,0 +1,68 @@
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+
+        CREATE EXTENSION if not exists "uuid-ossp";
+
+        ALTER TABLE "OrganizationEmployee" RENAME COLUMN "id" TO "old_id";
+        ALTER TABLE "OrganizationEmployee" ADD COLUMN "id" UUID NULL;
+
+        UPDATE "OrganizationEmployee" SET "id" = uuid_generate_v4();
+
+        ALTER TABLE "OrganizationEmployee" DROP CONSTRAINT "OrganizationEmployee_pkey";
+        ALTER TABLE "OrganizationEmployee" ADD PRIMARY KEY ("id");
+
+        ALTER TABLE "OrganizationEmployeeHistoryRecord" RENAME COLUMN "history_id" TO "old_history_id";
+        ALTER TABLE "OrganizationEmployeeHistoryRecord" ADD COLUMN "history_id" UUID NULL;
+
+        UPDATE "OrganizationEmployeeHistoryRecord" hr
+        SET "history_id" = e."id"
+        FROM "OrganizationEmployee" as e
+        WHERE(
+        e."old_id" = hr."old_history_id"
+        );
+
+        ALTER TABLE "OrganizationEmployeeHistoryRecord" ALTER COLUMN "history_id" SET NOT NULL;
+        ALTER TABLE "OrganizationEmployeeHistoryRecord" ALTER COLUMN "old_history_id" DROP NOT NULL;
+
+        ALTER TABLE "OrganizationEmployee" RENAME COLUMN "newId" TO "old_newId";
+        ALTER TABLE "OrganizationEmployee" ADD COLUMN "newId" UUID NULL;
+
+        COMMIT;
+    END;
+    `)
+}
+
+exports.down = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+        ALTER TABLE "OrganizationEmployee" RENAME COLUMN "id" TO "_old_id";
+        ALTER TABLE "OrganizationEmployee" RENAME COLUMN "old_id" TO "id";
+
+        ALTER TABLE "OrganizationEmployeeHistoryRecord" RENAME COLUMN "history_id" TO "_old_history_id";
+        ALTER TABLE "OrganizationEmployeeHistoryRecord" RENAME COLUMN "old_history_id" TO "history_id";
+        ALTER TABLE "OrganizationEmployeeHistoryRecord" ALTER COLUMN "history_id" SET NOT NULL;
+
+        ALTER TABLE "OrganizationEmployee" RENAME COLUMN "newId" TO "_old_newId";
+        ALTER TABLE "OrganizationEmployee" RENAME COLUMN "old_newId" TO "newId";
+        ALTER TABLE "OrganizationEmployee" DROP COLUMN "_old_newId";
+
+        UPDATE "OrganizationEmployeeHistoryRecord" hr
+        SET "history_id" = e."id"
+        FROM "OrganizationEmployee" as e
+        WHERE(
+        e."_old_id" = hr."_old_history_id" AND
+        "history_id" IS NULL

```

Comment:
Indent statements in `WHERE`:
```suggestion
            e."_old_id" = hr."_old_history_id" AND
            "history_id" IS NULL

```